ethereum/aave/insert_aave_daily_atoken_balances.sql

```diff
@@ -0,0 +1,136 @@
+CREATE OR REPLACE FUNCTION aave.insert_aave_daily_atoken_balances(start_time timestamptz, end_time timestamptz) RETURNS integer
+LANGUAGE plpgsql AS $function$
+DECLARE r integer;
+	start_time_day timestamptz := DATE_TRUNC('day',start_time);
+	end_time_day timestamptz := DATE_TRUNC('day',end_time) + interval '1 day'; --since we trunc to day
+BEGIN
+WITH rows AS (
+    INSERT INTO aave.aave_daily_atoken_balances (
+	day,
+	token_address,
+	daily_change,
+	starting_balance,
+	interest_rate_apr,
+	interest_earned,
+	total_balance
+    )
+	
+WITH RECURSIVE 
+gs AS --doing a series so that we don't skip days when there's no deposits/withraws
+(
+SELECT DISTINCT d.token_address AS token_address, gs.day FROM aave."aave_tokens" d
+INNER JOIN
+(SELECT generate_series(start_time_day, end_time_day, '1 day') AS day) gs -- Generate all days since the first contract
+ON 1=1
+)
+
+, rawbals AS ( --raw balances without interest. Doing these in a cte helps with runtime
+SELECT
+gs.day,gs."token_address" AS "contract_address", COALESCE(a.value,0) AS value,
+COALESCE("interest_rate_apr",0) AS interest_rate_apr, drank
+FROM (
+SELECT pre.*,
+DENSE_RANK() OVER (PARTITION BY pre."contract_address" ORDER BY DATE_TRUNC('day',pre."evt_block_time") ASC) AS drank
```

Comment:
This causes 
```
CREATE FUNCTION
psql:ethereum/aave/insert_aave_daily_atoken_balances.sql:127: ERROR:  column pre.evt_block_time does not exist
LINE 27: ... pre."contract_address" ORDER BY DATE_TRUNC('day',pre."evt_b...
                                                              ^
...                          
```

Reply:
Not sure if this should be `day` or if `evt_block_time` is missing below?

Reply:
This should be day, just fixed it!